packages/frontend/src/context/User.ts

```diff
@@ -106,6 +109,59 @@ export class User {
         return this.currentEpoch
     }
 
+    async loadRecords() {
+        const epksBase10 = this.allEpks.map((epk) =>
+            BigInt('0x' + epk).toString()
+        )
+
+        const apiURL = makeURL(`records/${epksBase10.join('_')}`, {})
+        const r = await fetch(apiURL)
+        const data = await r.json()
+        const rawRecords = data.map((r: Record) => {
```

Comment:
I would also add error handing for if the fetch request fails here. 

For example:
`if (!r) {  .... handle error ... }`

Reply:
oooops, sorry I forgot, thx for finding this :)